Plugins/Skinning/src/PositionBasedSkinning/PBS.hpp

```diff
@@ -0,0 +1,119 @@
+#ifndef RADIUMENGINE_POSITIONBASED_SKINNING_HPP
+#define RADIUMENGINE_POSITIONBASED_SKINNING_HPP
+#include <Core/Animation/Skinning/SkinningData.hpp>
+#include <Core/Math/LinearAlgebra.hpp>
+#include <Core/Animation/Handle/HandleWeight.hpp>
+#include "SimulationModel.h"
+
+namespace PositionBasedSkinning {
+
+class PBS {
```

Comment:
Add documentation on what's the purpose of the class, its methods and attributes